packages/botonic-react/src/components/multichannel/multichannel-text.jsx

```diff
@@ -82,7 +82,12 @@ export const MultichannelText = props => {
     if (multichannelContext.messageSeparator != null) {
       return elements
     }
-    return <Text {...props}>{elements}</Text>
+    return (
+      // We don't want to handle markdown here as this will be done internally by WhatsApp
```

Comment:
whatsapp does not support markdown. They have their own format which is incompatible most times even for bold and italics. 
https://faq.whatsapp.com/en/android/26000002/
See https://github.com/hubtype/botonic/blob/master/packages/botonic-plugin-contentful/src/markup/whatsapp.ts how I did the conversion in contentful plugin.
I suggest documenting the liimitation and leaving the conversion for another PR

Reply:
Okay, we will need to take a look on this, I have created a task for this in Linear 👍